creator-node/src/utils.js

```diff
@@ -74,13 +74,13 @@ async function getIPFSPeerId (ipfs, config) {
 }
 
 /** Cat single byte of file at given filepath. */
-const ipfsSingleByteCat = (path, req) => new Promise(async (resolve, reject) => {
+const ipfsSingleByteCat = (path, req, timeout = 1000) => new Promise(async (resolve, reject) => {
```

Comment:
it's not clear from code that the `ipfs.cat()` fn throws if the timeout is exceeded. a comment to that effect would be helpful